### DISCUSSION TITLE
Voice commands map pan, zoom, etc

### DISCUSSION THREAD

**Discussion by ninmorfeo** [osmandapp/OsmAnd#18893] [Ideas]
One of the biggest limitations of all the navigators currently out there is that you have to use your hands to use the map. It would be a wonderful thing if you could retire touch screen gloves. Maybe it is not yet possible to do this to use all the navigator functions but it is probably already possible to do so to use only the map.
Using voice commands (perhaps with artificial intelligence) such as "pan to the right 10 meters" or "center the map" or "zoom out 100 meters" etc. would be very convenient and would take the navigator to another level.

**Comment by Sonwon1** [comment]
I wonder if Android already has voice command software? I'll google that when I can.

**Comment by ZeljkoBG** [comment]
You're wasting time. iGO and Sygic developers tried, of course, via Google Box. They soon abandoned development. Far future.

**Reply by Sonwon1** [reply]
Maybe not, I asked over in Reddit and was told I need,

Tasker, Auto Voice and Auto Input

I will give it a go when I have more time.
